doc/source/learning/overview.rst

```diff
@@ -0,0 +1,38 @@
+Overview
+=====================
+
+This part of the documentation will help new developers to understand what is Airship and all its
+ecosystem of tools.
+
+This provides links to documentation and videos to help learning, but also to as an entry point to
+find the important documentation.
+
+First let's see how the Airship parts are going to work together.
+
+
+What is Airship?
++++++++++++++++++
+
+Airship is a platform that takes YAML documents and turns them into Clouds.
+
+In other workds, Airship is a set of tools that work together to help the operators and developers
+to create kubernetes clouds with a functional openstack. In socok8s is going implemented over CaaSP.
```

Comment:
We should mention that because of this (CaaSP requirement and independent deployment) we are not using airship's tools for setting up kubernetes cluster (which you are describing bellow)

Reply:
This part has been removed. See @evrardjp comment